app/config/storage.js

```diff
@@ -4,7 +4,7 @@ const Joi = require('joi')
 const schema = Joi.object({
   connectionStr: Joi.string().when('useConnectionStr', { is: true, then: Joi.required(), otherwise: Joi.allow('').optional() }),
   storageAccount: Joi.string().required(),
-  container: Joi.string().default('statement'),
+  container: Joi.string().default('statements'),
```

Comment:
Correcting name of container so consistent with higher environments and publisher repo

Reply:
Does this have knock-on effect as is, i.e. w/ those deployed in higher envs?

What is the remitting actions if so?

Reply:
It doesn't have any impact in higher environments.  These are just the default values used if none are provided through the environment variable.  For local development we don't set that env var so the defaults are used.

In higher envs, we're always deploying this service to Kubernetes through the Helm chart.  The Helm chart does set the env var through the config map and the value there was already the correct value of `statements`.